src/GitHub.VisualStudio/UI/Views/Controls/LoginControl.xaml.cs

```diff
@@ -39,7 +43,10 @@ public LoginControl()
                         NotifyDone();
                     }
                 });
+
+                d(Disposable.Create(() => errorHandler.Dispose()));
```

Comment:
Can `errorHandler` be null here? It's only set in the selection changed event below, and you check for nullness there, is that event guaranteed to be invoked always?